ppl/cmd/zar/README.md

```diff
@@ -433,43 +453,43 @@ run the following commands to a create a multi-key search index
 keyed by all unique instances of IP address pairs (in both directions)
 where each index includes a count of the occurrences.
 ```
-zar map -o forward.zng "id.orig_h != null | put from=id.orig_h,to=id.resp_h | count() by from,to"
-zar map -o reverse.zng "id.orig_h != null | put from=id.resp_h,to=id.orig_h | count() by from,to"
-zar map -o directed-pairs.zng "count=sum(count) by from,to | sort from,to" forward.zng reverse.zng
-zar index -i directed-pairs.zng -o graph.zng -k from,to -z "*"
+zar map -o forward.zng "id.orig_h != null | put from_addr=id.orig_h,to_addr=id.resp_h | count() by from_addr,to_addr"
+zar map -o reverse.zng "id.orig_h != null | put from_addr=id.resp_h,to_addr=id.orig_h | count() by from_addr,to_addr"
+zar map -o directed-pairs.zng "count=sum(count) by from_addr,to_addr | sort from_addr,to_addr" forward.zng reverse.zng
+zar index create -i directed-pairs.zng -o graph.zng -k from_addr,to_addr -z "*"
```

Comment:
I put in all this `_addr` stuff as a workaround to #2398. @mccanne pointed out that I could have also escaped this as `\from`, but I figured that we could just be more explicit rather than get into a tangent about what is hopefully a temporary workaround, or leaving the reader scratching their head wondering why we put that backslash in there.